Haulage/BaseClasses/Accounting/Address.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Haulage.BaseClasses.Accounting
+{
+    public class Address
+    {
+        public Address() { }
```

Comment:
This is currently a string in the UML diagram however should probably be expanded to a class containing address line 1, postcode etc (have created a issue for this)